lib/litmus_paper/dependency/base.rb

```diff
@@ -0,0 +1,16 @@
+module LitmusPaper
+  module Dependency
+    class Base
+      attr_reader :available
+
+      def available?
+        return @available unless @available.nil?
```

Comment:
@dkuntz2 You're right. I was thinking the dependency objects were reinitialized every health check, but it looks like they exist for the lifetime of the process. I added 2 commits to fix this.